swap/tests/e2e.rs

```diff
@@ -1,117 +1,124 @@
-#[cfg(not(feature = "tor"))]
-mod e2e_test {
-    use bitcoin_harness::Bitcoind;
-    use futures::{channel::mpsc, future::try_join};
-    use libp2p::Multiaddr;
-    use monero_harness::Monero;
-    use std::sync::Arc;
-    use swap::{alice, bob, network::transport::build};
-    use testcontainers::clients::Cli;
-    use tracing_subscriber::util::SubscriberInitExt;
-
-    #[tokio::test]
-    async fn swap() {
-        let _guard = tracing_subscriber::fmt()
-        .with_env_filter(
-            "swap=debug,xmr_btc=debug,hyper=off,reqwest=off,monero_harness=info,testcontainers=info,libp2p=debug",
-        )
+use bitcoin_harness::Bitcoind;
+use futures::{channel::mpsc, future::try_join};
+use libp2p::Multiaddr;
+use monero_harness::Monero;
+use std::sync::Arc;
+use swap::{alice, bob, network::transport::build, storage::Database};
+use tempfile::tempdir;
+use testcontainers::clients::Cli;
+use xmr_btc::bitcoin;
+
+// NOTE: For some reason running these tests overflows the stack. In order to
+// mitigate this run them with:
+//
+//     RUST_MIN_STACK=100000000 cargo test
```

Comment:
Add this to the readme please.

Reply:
This is no longer the case after #39 was merged.